deepchem/feat/tests/test_mxmnet_featurizer.py

```diff
@@ -0,0 +1,100 @@
+"""
+Test for MXMNet Featurizer class.
+"""
+
+from deepchem.feat.molecule_featurizers.mxmnet_featurizer import MXMNetFeaturizer
+import numpy as np
+import unittest
+
+edge_index = {
+    "C1=CC=NC=C1":
+        np.asarray([[0, 1, 1, 5, 5, 2, 2, 4, 4, 3, 3, 0],
+                    [1, 0, 5, 1, 2, 5, 4, 2, 3, 4, 0, 3]]),
+    "CC(=O)C":
+        np.asarray([[0, 3, 3, 2, 3, 1], [3, 0, 2, 3, 1, 3]]),
+    "C":
+        np.empty((2, 0), dtype=int)
+}
+
+node_features = {
+    "C1=CC=NC=C1": np.asarray([[1.], [1.], [2.], [1.], [1.], [1.]]),
+    "CC(=O)C": np.asarray([[1.], [1.], [3.], [1.]]),
+    "C": np.asarray([[1]])
```

Comment:
Here, the data type of feature is int and not float. Please check.

Reply:
yes corrected, Thank you.